[2014-03-20 18:01:14.409000]
ERROR
	Log observer <bound method FileLogObserver.emit of <twisted.python.log.FileLogObserver instance at 0x03027238>> failed.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\twisted\internet\selectreactor.py", line 155, in _doReadOrWrite
    self._disconnectSelectable(selectable, why, method=="doRead")
  File "D:\Python27\lib\site-packages\twisted\internet\posixbase.py", line 261, in _disconnectSelectable
    selectable.connectionLost(f)
  File "D:\Python27\lib\site-packages\twisted\internet\tcp.py", line 287, in connectionLost
    protocol.connectionLost(reason)
  File "E:\firefly\firefly\firefly\firefly\distributed\root.py", line 16, in connectionLost
    log.msg("node [%d] lose"%clientID)
--- <exception caught here> ---
  File "D:\Python27\lib\site-packages\twisted\python\log.py", line 205, in msg
    self.observers[i](actualEventDict)
  File "D:\Python27\lib\site-packages\twisted\python\log.py", line 402, in emit
    util.untilConcludes(self.flush)  # Hoorj!
  File "D:\Python27\lib\site-packages\twisted\python\util.py", line 776, in untilConcludes
    return f(*a, **kw)
exceptions.IOError: [Errno 22] Invalid argument

[2014-03-20 18:01:19.787000]
ERROR
	Log observer <bound method FileLogObserver.emit of <twisted.python.log.FileLogObserver instance at 0x03027238>> failed.
Traceback (most recent call last):
  File "D:\Python27\lib\site-packages\twisted\internet\selectreactor.py", line 155, in _doReadOrWrite
    self._disconnectSelectable(selectable, why, method=="doRead")
  File "D:\Python27\lib\site-packages\twisted\internet\posixbase.py", line 261, in _disconnectSelectable
    selectable.connectionLost(f)
  File "D:\Python27\lib\site-packages\twisted\internet\tcp.py", line 287, in connectionLost
    protocol.connectionLost(reason)
  File "E:\firefly\firefly\firefly\firefly\distributed\root.py", line 16, in connectionLost
    log.msg("node [%d] lose"%clientID)
--- <exception caught here> ---
  File "D:\Python27\lib\site-packages\twisted\python\log.py", line 205, in msg
    self.observers[i](actualEventDict)
  File "D:\Python27\lib\site-packages\twisted\python\log.py", line 402, in emit
    util.untilConcludes(self.flush)  # Hoorj!
  File "D:\Python27\lib\site-packages\twisted\python\util.py", line 776, in untilConcludes
    return f(*a, **kw)
exceptions.IOError: [Errno 22] Invalid argument

